google/ggrc-core
# QUICK-FIX Increase box size for ms ca on info modal

# Issue description

Box size for multiselect CA is now wide enough to display potentially long options.

# Steps to test the changes

 - Create GCA of multiselect type
 - Create object w/ multiselect GCA & fill out ms CA
 - Open created object's info pane & scroll down to CAs section

Expected behavior: Box size for multiselect CA is the same as for textable CA

![Screenshot from 2019-05-31 11-40-52](https://user-images.githubusercontent.com/16187978/58693440-781e5700-8399-11e9-8357-a5b1cfd63530.png)


# Solution description

Apply ```ggrc-form-item--row``` for multiselect CA type.

# Sanity checklist




# PR Review checklist
